src/llm/http_llm_calculator.cc

```diff
@@ -525,10 +142,7 @@ class HttpLLMCalculator : public CalculatorBase {
                     break;
                 }
                 case Endpoint::COMPLETIONS: {
-                    if (!this->request->getPrompt().has_value() || !this->request->getPrompt().value().size()) {
```

Comment:
Is this validation removed on purpose ?

Reply:
Same question above and below.

Reply:
It's been moved to openai_completions file to processRequest... methods